### DISCUSSION TITLE
Clear logs

### DISCUSSION THREAD

**Discussion by youngdesenv** [AzuraCast/AzuraCast#5627] [Q&A]
How do i clear the logs?

**Comment by ItsMitchh** [comment]
You cannot clear the logs natively as they're used for a variety of processes and functions, such as debugging and well, logging. Without the logs it'd be impossible to debug your machine or for you to see if there's any issues.

**Comment by youngdesenv** [comment]
But I have a slow system, even without high CPU or memory consumption and the HD is being used very little.

It seems to be something related to the log files being too big.

**Reply by BusterNeece** [reply]
@youngdesenv We've made some changes to how we process logs on the system to both prevent them from exceeding a given size and to prevent the CPU from overrunning when printing the logs. This should fix any issues associated with system logging.

**Comment by tomy91** [comment]
1. Connect to AzuraCast server via SSL (Putty)
2. Enter the docker container `docker exec -it azuracast bash`
3. Navigate to AzuraCast log directory of the station `cd /var/azuracast/stations/<your station folder>/config`
4. Use `truncate -s 0 *.log` to clear all logs or change * to a filename of a specific log file.
5. Profit.
